scripts-dev/mypy_synapse_plugin.py

```diff
@@ -68,6 +68,18 @@
     category="per-homeserver-tenant-metrics",
 )
 
+INTERNAL_CLOCK_CALL_LATER_NOT_USED = ErrorCode(
+    "call-later-not-tracked",
+    "`synapse.util.Clock.call_later` should be used instead of `reactor.callLater`",
+    category="delayed-call-tracking",
+)
+
+INTERNAL_CLOCK_LOOPING_CALL_NOT_USED = ErrorCode(
+    "looping-call-not-tracked",
```

Comment:
Perhaps this makes more sense:

```suggestion
    "prefer-synapse-clock-call-later",
```